src/manifest/extensions/editor/EditorRegion.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars */
+
+export enum EditorRegion {
+  /**
+   * Template context host
+   */
+  TEMPLATE = 'template',
+
+  /**
+   * Snippet context host
+   */
+  SNIPPET = 'snippet',
+
+  /**
+   * One off email context host
+   */
+  ONE_OFF_EMAIL = 'oneoffEmail',
+
+  /**
+   * Bulk email context host
+   */
+  BULK_EMAIL = 'bulkEmail',
+
+  /**
+   * Task flow context host
+   */
+  TASK_FLOW = 'taskFlow',
+
+  /**
+   * Sequence editor context host
+   */
+  SEQUENCE = 'sequenceEditor',
+
+  /**
+   * Other context host
+   */
+  OTHER = 'other',
```

Comment:
Why do we need `OTHER` ?

Reply:
It is Galaxy enum values used by the existing text editors - not sure in which exactly place of app is used